File: Uri 1002/Main.java
//@author Diego Lins

import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		
		Scanner in =new Scanner(System.in);
	    double raio=in.nextDouble();
	    double n = 3.14159;
	    double area = n * Math.pow(raio, 2);
	    System.out.printf("A=%.4f%n", area);	
		
	}

}

File: Uri 1002/Main.js
//@Author Diego Lins

var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var r = parseFloat(lines.shift());
var pi = 3.14159;
var a = pi * (Math.pow(r,2));
console.log("A=" + a.toFixed(4));